CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 3.0.0-rc.21 (2019-10-02)
+- Fix renderTheme function in order to not lose templates in stencil cli [#179](https://github.com/bigcommerce/paper/pull/179)
```

Comment:
Can we also add there is bump for paper handlebars

Reply:
fixed.